docs/en/security/flash-encryption.rst

```diff
@@ -413,7 +413,7 @@ To use a host generated key, take the following steps:
 
         idf.py --port PORT efuse-burn-key BLOCK my_flash_encryption_key.bin KEYPURPOSE
 
-    where ``BLOCK`` is a free keyblock between ``BLOCK_KEY0`` and ``BLOCK_KEY5``. And ``KEYPURPOSE`` is either ``AES_256_KEY_1``, ``XTS_AES_256_KEY_2``, ``XTS_AES_128_KEY``. See `{IDF_TARGET_NAME} Technical Reference Manual <{IDF_TARGET_TRM_EN_URL}>`_ for a description of the key purposes.
+    where ``BLOCK`` is a free keyblock between ``BLOCK_KEY0`` and ``BLOCK_KEY5``. And ``KEYPURPOSE`` is either ``XTS_AES_256_KEY_1``, ``XTS_AES_256_KEY_2``, ``XTS_AES_128_KEY``. See `{IDF_TARGET_NAME} Technical Reference Manual <{IDF_TARGET_TRM_EN_URL}>`_ for a description of the key purposes.
```

Comment:
@kindaTall Can you please fix this in `docs/zh_CN/security/flash-encryption.rst` guide as well?

Reply:
@mahavirj Done :)

Reply:
@kindaTall Thanks. Can you please squash the commits and change commit title to:

```
docs(security): fix typo for key purpose field in flash encryption guide
```

Reply:
@mahavirj Should be done. First time doing this all in browser. sorry for the delay.